llrb/llrb.go

```diff
@@ -19,6 +19,7 @@ package llrb
 // Tree is a Left-Leaning Red-Black (LLRB) implementation of 2-3 trees
 type LLRB struct {
 	count int
+	cow   bool
```

Comment:
unfortunately a LLRBCoW type which embeds LLRB doesn't work to alter the behavior of walkDownRot23 because it's called internally from the various insert methods.